dbms/src/Flash/tests/gtest_aggregation_executor.cpp

```diff
@@ -333,6 +343,20 @@ try
 }
 CATCH
 
+TEST_F(ExecutorAggTestRunner, AggregationSum)
+try
+{
+    // Test output type of the first and second stage sum aggregation function
+    PrecType precision = 33;
+    ScaleType scale = 7;
+    std::shared_ptr<IDataType> input_type_ptr1 = std::make_shared<DataTypeDecimal<Decimal64>>(precision, scale);
+    std::shared_ptr<IDataType> expect_output_type_ptr1 = std::make_shared<DataTypeDecimal<Decimal64>>(precision, scale);
+    EXPECT_TRUE(checkReturnType(NameFinalSumStage::name, {input_type_ptr1}, expect_output_type_ptr1));
+
+    // TODO Test sum aggregation
+}
+CATCH
```

Comment:
I think it's better to move to a new test file.

Reply:
Because this is not a test for Agg Executor

Reply:
> I think it's better to move to a new test file.

sure